Library/Homebrew/cmd/man.rb

```diff
@@ -25,21 +25,31 @@ def man
       puts "Writing HTML fragments to #{DOC_PATH}"
       puts "Writing manpages to #{TARGET_PATH}"
 
-      Dir["#{SOURCE_PATH}/*.md"].each do |source_file|
-        args = %W[
-          --pipe
-          --organization=Homebrew
-          --manual=brew
-          #{source_file}
-        ]
-        page = File.basename(source_file, ".md")
-
-        target_html = DOC_PATH/"#{page}.html"
-        target_html.atomic_write Utils.popen_read("ronn", "--fragment", *args)
-
-        target_man = TARGET_PATH/page
-        target_man.atomic_write Utils.popen_read("ronn", "--roff", *args)
-      end
+      header = IO.read("#{SOURCE_PATH}/header.1.md")
+      footer = IO.read("#{SOURCE_PATH}/footer.1.md")
+      sub_commands = Dir["#{HOMEBREW_REPOSITORY}/Library/Homebrew/cmd/*"].sort.map { |source_file|
+        IO.read(source_file).
+          split("\n").
+          grep(/^#:/).
+          map { |line| line.slice(2..-1) }.
+          join("\n")
+      }.reject { |s| s.strip.empty? }.join("\n\n")
+
+      IO.write("#{SOURCE_PATH}/brew.1.md", header + sub_commands + footer)
+
+      page = "brew.1"
```

Comment:
For now we're only generating one man page, so let's get rid of the variable `page` completely and substitute the literal `"brew.1"` in places where it's currently still used.